RocketChat/Rocket.Chat
# [IMPROVE] VideoConf Miss Config Modal

## Proposed changes (including videos or screenshots)


Jira Issue: [VC-9]

<img width='420px' src='https://user-images.githubusercontent.com/27704687/199082670-7a5b47ea-077b-48f1-b232-ccdfa682e087.png' />



## Issue(s)


## Steps to test or reproduce

- With no video apps activated try to initiate a call 
- The modal with the feature content will appear
- There are differences in copy for admin and regular users

## Further comments



[VC-9]: https://rocketchat.atlassian.net/browse/VC-9?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
